transformers/storage/repository.go

```diff
@@ -66,6 +68,19 @@ func (repository *MakerStorageRepository) GetClipSalesIDs(contractAddress string
 	return saleIDs, err
 }
 
+func (repository *MakerStorageRepository) GetActiveLength(contractAddress string) (string, error) {
+	var activeLength sql.NullString
+	addressID, addressErr := repository.GetOrCreateAddress(contractAddress)
+	if addressErr != nil {
+		return "", addressErr
```

Comment:
Not sure how I feel about returning the empty string here in the case of an address error.

Reply:
Does zero make more sense?